shell.nix

```diff
@@ -33,8 +33,8 @@ let
     tools = {
       cabal = "3.8.1.0";
       ghcid = "0.8.7";
-      ormolu = "0.5.0.1";
-      haskell-language-server="latest";
+      fourmolu = "0.10.1.0";
```

Comment:
so was this already in the nix store, or is this related to you changing the nix sources?

Reply:
I updated nix sources in order to get the latest version of fourmolu, cause `0.10.1.0` is a fairly recent release